Documentation for Notebooks (#6971)
PostHog/posthog.com: contents/docs/notebooks/index.mdx

You can get started with notebooks by going to [the notebook tab](https://app.posthog.com/notebooks) in your PostHog instance and clicking "New notebook."

## Accessing the notebook popover

Critical to combining the functionality of PostHog is being able to access notebooks from anywhere, like a scratchpad for analysis which you can organize and clean up later.

You can access a notebook by clicking the journal icon in the navbar (next to the notification bell) or by clicking and dragging a link anywhere in PostHog.

![Popover icon](../../images/docs/notebooks/popover.png)

To switch the notebook that opens in the popover, click the title button in the top left of the popover 

![Switch popover notebook](../../images/docs/notebooks/switch.png)

## Adding content to notebooks

To start adding content to notebooks, start typing. There are three main ways to add content to notebooks:

1. **Slash commands.** Within a notebook, type `/` and a menu will pop up with options for you. For example, `/trend` creates a trend insight.

2. **Click the `+` icon on an empty line**. This opens the same menu as slash commands.

> 3. **Dragging to popover.** From the other sections in PostHog, click and hold a link and a popover notebook shows up on the right side you can drag into. 

Reviewer: I feel like these would for sure benefit from gifs / screenshots.

Reviewer 2: Added one for each.

You can rearrange content in notebooks by dragging and dropping components into a different order.

## Notebook content types

### Text

Notebooks supports the standard text editing features via Markdown. These include:

- \# Heading 1
- \## Heading 2
- \### Heading 3
- \- List

> - `1.` Numbered list

Reviewer: We support check boxes now!

```suggestion
- `1.` Numbered list
- [] Check list
```

---
title: Notebooks
sidebar: Docs
showTitle: true
availability:
    free: full
    selfServe: full
    enterprise: full
---

> > Notebooks are currently in **public beta**. This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/15680)!

Author: ```suggestion
> [Notebooks are currently in **public beta**](/docs/getting-started/enable-betas). This means it's not yet a perfect experience, but we'd love to know your thoughts. Please [share your feedback](http://app.posthog.com/home#supportModal) and [follow our roadmap](https://github.com/PostHog/posthog/issues/15680)!
```

Notebooks enable you to combine the tools of PostHog into a single space for analysis. Rather than jumping between dashboards, insights, replays, experiment results, and more, your chain of analysis exists on a single page.

> It provides a flexible interface for organizing ad-hoc analysis, bug investigations, internal documentation, and feature releases by yourself or with your time.

Reviewer 3: ```suggestion
It provides a flexible interface for organizing ad-hoc analysis, bug investigations, internal documentation, and feature releases by yourself or with your team.
```

You can get started with notebooks by going to [the notebook tab](https://app.posthog.com/notebooks) in your PostHog instance and clicking "New notebook."

## Accessing the notebook popover

Critical to combining the functionality of PostHog is being able to access notebooks from anywhere, like a scratchpad for analysis which you can organize and clean up later.

You can access a notebook by clicking the journal icon in the navbar (next to the notification bell) or by clicking and dragging a link anywhere in PostHog.

![Popover icon](../../images/docs/notebooks/popover.png)

> To switch the notebook that opens in the popover, click the title button in the top left of the popover 

Reviewer 2: ```suggestion
To switch the notebook that opens in the popover, click the title button in the top left of the popover.
```

![Switch popover notebook](../../images/docs/notebooks/switch.png)

## Adding content to notebooks

To start adding content to notebooks, start typing. There are three main ways to add content to notebooks:

1. **Slash commands.** Within a notebook, type `/` and a menu will pop up with options for you. For example, `/trend` creates a trend insight.

![Slash command](../../images/docs/notebooks/slash.mp4)

2. **Click the `+` icon on an empty line**. This opens the same menu as slash commands.

![Plus icon](../../images/docs/notebooks/plus.mp4)

3. **Dragging to popover.** From the other sections in PostHog, click and hold a link and a popover notebook shows up on the right side you can drag into. 

![Drag and drop](../../images/docs/notebooks/drag.mp4)

You can rearrange content in notebooks by dragging and dropping components into a different order.

## Notebook content types

### Text

Notebooks supports the standard text editing features via Markdown. These include:

> 

Reviewer 2: ```suggestion
```

Once you add a feature flag to your notebook, you can see its status, release conditions, implementation code, and related session replays. Using it, you can create surveys and early access features. 

![Flag](../../images/docs/notebooks/flag.png)

### Surveys

You can add a survey by dragging and dropping its link into your notebook. 

Once you add one, you can see its status, views, dismissals, and submissions. Depending on the survey type, you can also see more details or visualizations for the responses. 

![Survey](../../images/docs/notebooks/survey.png)

### Images

You can add both images (.png, .jpg, .jpeg, .bmp) and GIFs to your notebooks using the slash command or by dragging and dropping. 

### Links

You can create preview links for areas within PostHog, like dashboards and notebooks, by dragging and dropping them into notebooks.

You can create external links by highlighting text and then pasting the link in the box.

## Frequently asked questions

### How much do notebooks cost?

Notebooks are free. [Sign up for free](https://app.posthog.com/signup) to try them out.

### What are the differences between dashboards and notebooks?

> Notebook incorporating all the components of PostHog. Dashboards specifically focus on analytics, insights, and events. 

Reviewer 3: ```suggestion
Notebooks incorporate all the components of PostHog. Dashboards specifically focus on analytics, insights, and events. 
```
